src/imputors/drop.jl

```diff
@@ -33,7 +33,7 @@ function impute(data::Vector{<:NamedTuple}, imp::DropObs)
     return filter(r -> all(!ismissing, propertyvalues(r)), data)
 end
 
-function impute(data::AbstractMatrix, imp::DropObs; dims=1)
+function impute(data::AbstractMatrix{Union{T, Missing}}, imp::DropObs; dims=1) where T
```

Comment:
I like this change a lot.
Do we provide a `imput!(data::AbstractMatrix,...) = data`
and `imput(data::AbstractMatrix,...) = copy(data)`